nucliadb_utils/src/nucliadb_utils/authentication.py

```diff
@@ -32,8 +32,9 @@
 
 
 class NucliaUser(BaseUser):
-    def __init__(self, username: str) -> None:
+    def __init__(self, username: str, security_groups: Optional[list[str]] = None) -> None:
         self.username = username
+        self.security_groups = security_groups
```

Comment:
As endpoints will directly access this field, maybe we should use a read only property instead

Reply:
sure